py3status/modules/spotify.py

```diff
@@ -27,12 +27,23 @@
     color_paused: Song is stopped or paused, defaults to color_degraded
     color_playing: Song is playing, defaults to color_good
 
+Click Events:
+    You can add some config to your i3status.conf to  create click events
+    For example
```

Comment:
* We can get rid of this section in favor of `Configuration parameters:` docs for the buttons.
* This PR is missing the buttons for the `Configuration parameters:` docs.
* This PR is missing the button variables in the `available configuration parameters` section. 
* See `cmus` / `moc` if you want to do some quick copy and paste for the docs.